Sources/AblyAssetTrackingSubscriber/WorkerQueue/SubscriberWorkerQueueProperties.swift

```diff
@@ -1,7 +1,181 @@
 import Foundation
 import AblyAssetTrackingInternal
 
-struct SubscriberWorkerQueueProperties: WorkerQueueProperties
-{
+struct SubscriberWorkerQueueProperties: WorkerQueueProperties {
     public var isStopped = false
+    var presenceData: PresenceData
+    let ablySubscriber: Subscriber
+    
+    private var updatingResolutions: [String: [Resolution?]] = [:]
```

Comment:
It would be helpful to know which version of the Android codebase you have based your work on. `updatingResolutions` doesn't exist in `v1.6.3`, which is what we're basing our current work on.

Reply:
Yeah, that's my mistake - used one of the newest commits as reference, forgot we're supposed to stick with 1.6.3. I'll rewrite it to the 1.6.3 state.